template.yaml

```diff
@@ -35,6 +35,7 @@ Parameters:
   CustomDomainBasePath:
     Type: String
     Description: Base path mapping in CustomDomain
+    Default: doi-fetch
```

Comment:
write default path of the service as code, so that the service "owns" its path. Can still be overridden if necessary, but we do not have to define the default case manually every time we deploy the service